### PR TITLE
[FEATURE] use config.json as source for app directory

### PR DESCRIPTION
**Pull request by sidohaakma**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by sidohaakma** [comment]
test this please

File: molgenis-app-manager/src/main/java/org/molgenis/app/manager/service/impl/AppManagerServiceImpl.java [hunk 1]
```diff
@@ -170,16 +200,23 @@ public void uploadApp(InputStream zipData, String zipFileName, String formFieldN
...
-		newApp.setResourceFolder(appDirectoryName);
+		newApp.setResourceFolder(appDirName);
 		newApp.setAppConfig(gson.toJson(runtimeOptions));
 		newApp.setUri(appConfig.getUri());
 
 		dataService.add(AppMetadata.APP, newApp);
 	}
 
+	@Override
+	public String extractFileContent(String appDir, String fileName)
```
Comment (ConnorStroomberg) [inline molgenis-app-manager/src/main/java/org/molgenis/app/manager/service/impl/AppManagerServiceImpl.java]: this public method is not covered by unit tests

File: molgenis-data-file/src/main/java/org/molgenis/data/file/FileStore.java [hunk 2]
```diff
@@ -46,6 +48,30 @@ public File store(InputStream is, String fileName) throws IOException
 		return file;
 	}
 
+	/**
+	 * Move directories in FileStore
+	 * Pleae provide the path from the relative root of the fileStore
```
Comment (ConnorStroomberg) [inline molgenis-data-file/src/main/java/org/molgenis/data/file/FileStore.java]: typo 'Pleae' , not need to 'please' there just keep it short and to the point

File: molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppManagerController.java [hunk 3]
```diff
@@ -71,6 +74,11 @@ public void uploadApp(@RequestParam("file") MultipartFile multipartFile) throws
 		InputStream fileInputStream = multipartFile.getInputStream();
 		String filename = multipartFile.getOriginalFilename();
 		String formFieldName = multipartFile.getName();
-		appManagerService.uploadApp(fileInputStream, filename, formFieldName);
+		String tempDir = appManagerService.uploadApp(fileInputStream, filename, formFieldName);
```
Comment (ConnorStroomberg) [inline molgenis-app-manager/src/main/java/org/molgenis/app/manager/controller/AppManagerController.java]: several of these calls throw a io exception, i would think we would need to handle this

**Comment by sidohaakma** [comment]
test this please

**Pull request merged**